addons/web/static/src/legacy/scss/web_calendar.scss

```diff
@@ -576,6 +576,9 @@ $o-cw-filter-avatar-size: 20px;
         // sync buttons are only displayed on calendar.event views
         .o_calendar_sync {
             padding-top: 0.5em;
+            span {
```

Comment:
Why this change by the way ?

Reply:
I'm adding text and need its size to be consistent with the surrounding elements. This effectively makes it slightly smaller.

Reply:
As discussed: trying the bootstrap fs class ?